layouts/New/Centered.tsx

```diff
@@ -0,0 +1,22 @@
+import type { FC, PropsWithChildren } from 'react';
+
+import WithFooter from '@/components/withFooter';
+import WithNavBar from '@/components/withNavBar';
+
+import styles from './layouts.module.css';
+
+const CenteredLayout: FC<PropsWithChildren> = ({ children }) => (
+  <>
+    <WithNavBar />
+
+    <div className={styles.centeredLayout}>
+      <div className="glowingBackdrop" />
```

Comment:
I'd say that this shouldn't be part of the centered layout itself, but since as from now all use cases have it we can leave it here.